src/store/transaction.js

```diff
@@ -146,6 +148,7 @@ export default {
     aggregateCosignatures: (state, aggregateCosignatures) => Vue.set(state, 'aggregateCosignatures', aggregateCosignatures),
     pageListFormatted: (state, pageListFormatted) => Vue.set(state, 'pageListFormatted', pageListFormatted),
     transactionInfoLoading: (state, v) => state.transactionInfoLoading = v,
+    transactionInfoError: (state, v) => state.transactionInfoError = v,
```

Comment:
should we use `Vue.set` to make everything synchronize?

Reply:
When we add a new object property, it is important to use Vue.set to make it reactive. `transactionInfoError` is already reactive and we change only it's value, so we can use `=` operator